ccmlib/scylla_repository.py

```diff
@@ -234,15 +235,40 @@ def setup(version, verbose=True, skip_downloads=False):
               - release:4.2.1
               - release:2020.1
               - release:2020.1.5
+            flags can be added to the version using [(flag(=value)?)+]
+              - supported flags are:
+                - product=(scylla|scylla-enterprise)
+                - arch=(x86_64|aarch64)
```

Comment:
Is there an use case where one would want a different arch than that of their system's?

Reply:
I think it could be good to support installing scylla from a foreign architecture package and launch it remotely on an appropriate machine.  I don't think it's done in practice now though.

Reply:
In theory on can download the arm files with it in you machine while running the tests under docker with emulation of different arch then your machine.

In practice we call install.sh, and it can't work on the wrong arch.

So I don't think there is such a case for now